test/test-framework/pkg/apis/cache/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -1,6 +1,6 @@
 // +build !ignore_autogenerated
 
-// Code generated by deepcopy-gen. DO NOT EDIT.
+// Code generated by operator-sdk. DO NOT EDIT.
```

Comment:
Did our code generate this output with our binary name, or is this a manual change? We should not edit these files at all after the generators have run.

Reply:
No. Since the Dockerfile was added it don't need more be done manually. So, this output came from the code source of operator-sdk. All here was updated and generated by running `operator-sdk generate [openapi, k8s]` commands.